web/src/compose_validate.js

```diff
@@ -25,6 +25,12 @@ let wildcard_mention;
 
 export let wildcard_mention_large_stream_threshold = 15;
 
+export function get_compose_banner_container($textarea) {
+    return $textarea.attr("id") === "compose-textarea"
+        ? $("#compose_banners")
+        : $textarea.closest(".message_edit_form").find(".edit_form_banners");
+}
```

Comment:
This function should be in `compose_banner`. Think about abstractions -- this is logic about the compose banners, not the validation system, and so it belongs there.

Reply:
Agree! Moved this into `compose_banner`.